src/MapKit/MKEnums.cs

```diff
@@ -140,6 +143,24 @@ public enum MKFeatureVisibility : long {
 		Hidden,
 		Visible,
 	}
+
+	[TV (13,0), NoWatch, Mac (10,15), iOS (13,0)]
+	[Native]
+	public enum MKLocalSearchCompleterResultType : long
```

Comment:
the use of `<<` suggest this is used with binary operators so it should
* be unsigned,`ulong`, and that's how my sharpie's output looks like;
* have a `[Flags]` attribute